### PR TITLE
fix(ngc): comment out a private keyword in codegen.

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

File: modules/@angular/compiler/src/output/ts_emitter.ts [hunk 1]
```diff
@@ -129,7 +129,8 @@ class _TsEmitterVisitor extends AbstractEmitterVisitor implements o.TypeVisitor
 
   private _visitClassField(field: o.ClassField, ctx: EmitterVisitorContext) {
     if (field.hasModifier(o.StmtModifier.Private)) {
-      ctx.print(`private `);
+      // comment out as a workaround for b/30775898
```
Comment (vikerman) [inline modules/@angular/compiler/src/output/ts_emitter.ts]: Please refer to external github issue at https://github.com/angular/angular/issues/10967 - repros with @ViewChildren.

**Comment by vikerman** [comment]
LGTM

**Comment by alexeagle** [comment]
I believe @vikerman or @robwormald has found the root cause and has a proper fix, closing.

**Comment by alexeagle** [comment]
We have a root cause but not a proper fix. Going ahead with this workaround for now.

**Pull request merged**